ansible_collections/ds/ansible_ds/tests/README.md

```diff
@@ -0,0 +1,16 @@
+389 Directory Server Ansible collection tests
+==========================
+
+This directory contains the test framework
+
+Note: tests are either:
+    - test_xxxx methods within python file or 
+    - test_xxxx.yml playbooks
+
+They can be run:
+    - directly from pytest. (Usefull when wanting to run a single test: i.e pytest $PWD -k test_gather_dsinst_info.yml)
+    - through ansible-test i(i.e: ansible-test units)
+    - through make (i.e: make unit_test)
```

Comment:
I think it can be skipped here as it refers to `../../../../` part and if we run it from this directory, it'll fail.